launch/vn_100_cont.launch

```diff
@@ -1,40 +1,31 @@
 <launch>
-    <!-- Node Settings -->
     <arg name="output" default="screen"/>
     <arg name="imu" default="imu"/>
 
-    <!-- IMU Settings -->
     <arg name="port" default="/dev/ttyUSB0" />
-
-    <!-- Frame ID for messages -->
     <arg name="frame_id" default="$(arg imu)"/>
-
-    <!-- Baudrate of serial comms (see manual for allowed values) -->
     <arg name="baudrate" default="921600"/>
-
-    <!-- Data rate in Hz -->
-    <arg name="imu_rate" default="200"/>
+    <arg name="imu_rate" default="100"/>
 
```

Comment:
Maybe add the binary output enabled param here, just for completeness. Either this or in the readme. Admittedly thoughI can't see why someone would want to get data in ascii.